captum/optim/_utils/image/common.py

```diff
@@ -1,10 +1,66 @@
 import math
 from typing import List, Optional, Tuple
 
+import matplotlib.pyplot as plt
+import numpy as np
 import torch
 
 from captum.optim._utils.reducer import posneg
 
+try:
+    from PIL import Image
+except (ImportError, AssertionError):
+    print("The Pillow/PIL library is required to use Captum's Optim library")
+
+
+def show(
```

Comment:
Are the `show` and `save_tensor_as_image` functions added to the `__init__.py` files anywhere? Because we should make sure that they are easily callable by the user.

Reply:
They are currently exposed as `optim.images.show` and `optim.images.save_tensor_as_image`.
Lmk if we want different behavior?